packages/docs/site/docs/01-start-here/03-repository-overview.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: /playground-ecosystem
+---
+
+# WordPress Playground ecosystem
+
+The various tools that make up the Playground project are managed in a few separate GitHub repositories:
+- [https://github.com/WordPress/wordpress-playground](https://github.com/WordPress/wordpress-playground/).
+- [https://github.com/WordPress/playground-tools](https://github.com/WordPress/playground-tools/).
+- [https://github.com/WordPress/blueprints](https://github.com/WordPress/blueprints).
+- [https://github.com/WordPress/blueprints-library](https://github.com/WordPress/blueprints-library/).
+
+By mapping where each part is, we hope to encourage you to get familiar with the codebase and understand how Playground is built. Visit the relevant repository to ask questions, report bugs, discover solutions to problems you run into, and share your knowledge with the community.
+
+## WordPress Playground
+
+The [main repository](https://github.com/WordPress/wordpress-playground) contains the core codebase and additional browser-based [utilities and demos](https://wordpress.github.io/wordpress-playground/links-and-resources#apps-built-with-wordpress-playground). Here are a few highlights:
+
+- [Blueprint Builder](https://playground.wordpress.net/builder/builder.html): create and test Blueprints in the browser ([source code](https://github.com/WordPress/wordpress-playground/tree/trunk/packages/playground/website/builder)).
+- [WordPress PR Previewer](https://playground.wordpress.net/wordpress.html): test pull requests submitted to WordPress core in the browser ([source code](https://github.com/WordPress/wordpress-playground/blob/trunk/packages/playground/website/public/wordpress.html)).
+- [Gutenberg PR Previewer](https://playground.wordpress.net/gutenberg.html): test pull requests from the Gutenberg plugin in the browser ([source code](https://github.com/WordPress/wordpress-playground/blob/trunk/packages/playground/website/public/gutenberg.html)).
+
+Explore the [source code for Playground](https://github.com/WordPress/wordpress-playground/tree/trunk/packages/playground), join [the discussions](https://github.com/WordPress/wordpress-playground/discussions), suggest improvements by opening [issues](https://github.com/WordPress/wordpress-playground/issues) (or help others by commenting on their questions), submit [PRs](https://github.com/WordPress/wordpress-playground/pulls) for bug fixes, contribute to the documentation, and be a part of the community.
+
+## WordPress Playground tools
```

Comment:
suggesting to remove "WordPress" from here on out, as the Playground becomes closer to Core it will become more clear that all of the features are part of the `WordPress Playground`

Reply:
That's the repo's [name](https://github.com/WordPress/playground-tools/blob/0902bf3a238c8969f12110875c4598239612cc7e/README.md?plain=1#L1).